packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -725,7 +725,7 @@ blockquote {
   position: fixed;
   width: 100%;
   z-index: 9999;
-  transform: translate3d(0,0,0);
+  transform: translate3d(0, 0, 0);
```

Comment:
Thanks for fixing all these style inconsistencies, but we'd prefer to keep them out of this PR as it makes it harder to review (could make another PR to address styling issues).

Reply:
Removed this and one more.